docs/guided-remediation.md

```diff
@@ -0,0 +1,247 @@
+
+## Interactive mode
+
+Interactive mode provides a step-by-step process to understand and fix vulnerabilities in your project.
+
+To run it, you can use the following command:
+
+```bash
+osv-scanner fix -M path/to/package.json -L path/to/package-lock.json
+```
+
+{: .warning }
+The subcommand will modify your manifest and lockfile. Make sure you commit or backup your files before running.
+
+The command will launch the Guided Remediation TUI and begin scanning your manifest and lockfile for vulnerabilities. Use the arrow keys to navigate menu items and the `ENTER` key to select. `TAB` can be used to switch to and from the information panel of each option.
+
+![Screenshot of the initial interactive results screen, showing found vulnerabilities and the potential remediation approaches](images/guided-remediation-result.png)
+
+{: .highlight }
+
+> The initial screen allows you to change the vulnerability selection criteria. Each correspond to a [vulnerability selection flag](#vulnerability-selection):
+>
+> - Max dependency depth: `--max-depth`
+> - Min CVSS score: `--min-severity`
+> - Exclude dev only: `--ignore-dev`
+
+From the first results screen, you can select which of the two remediation strategies to attempt.
+
+## In-place lockfile remediation
+
+'In-place' remediation involves replacing vulnerable versions of packages in your lockfile with non-vulnerable versions, while still respecting the existing constraints for that dependency. This approach is usually less risky, but will often fix less vulnerabilities than the [relock strategy](#relock-and-relax-direct-dependency-remediation).
+
+Selecting the "Modify lockfile in place" option will bring you to the in-place information page. From here, you can see which vulnerabilities can and cannot be resolved by this strategy. By default, every possible in-place patch will be chosen to be applied. You may instead choose which subset of patches you wish to apply.
+
+![Screenshot of the interactive in-place results screen](images/guided-remediation-in-place-results.png)
+
+![Screenshot of the interactive in-place patch selection screen](images/guided-remediation-in-place-choose.png)
+
+If you wish to apply the proposed in-place patches, select the "Write" option to update your lockfile with the new dependency versions.
+
+{: .note }
+Writing these changes will not reinstall your dependencies. You'll need to run `npm ci` (or equivalent) separately.
+
+## Relock and relax direct dependency remediation
+
+Relocking recomputes your entire dependency graph based on your manifest file, taking the newest possible versions of all your required packages. This causes a large number of changes to your dependency graph, which potentially carries a larger risk of breakages.
```

Comment:
Also clarify that these upgrades often unblock constraints allowing more vulnerabilities to be fixed.

Reply:
Added another sentence clarifying